src/server/Game.cpp

```diff
@@ -95,6 +95,7 @@ int Game::mainLoop() {
         if (elapsed_boss_seconds.count() > 20 && !bossSpawned) {
             Factory::Enemy::makeEndboss(this->_entManager);
             bossSpawned = true;
+            this->_protocol.sendChangeMusic(0);
```

Comment:
Since this is the only time that the server sends a change music, what change music does this represent ? Could you add a comment ?
Also, a little bit unrelated to this branch, but does the music loop by default.

Reply:
The music in raylib loops automaticly.